tests/checkpointing/test_checkpoint_callback_frequency.py

```diff
@@ -50,7 +50,7 @@ def test_mc_called(tmpdir):
 @mock.patch('torch.save')
 @pytest.mark.parametrize(
     ['epochs', 'val_check_interval', 'expected'],
-    [(1, 1.0, 1), (2, 1.0, 2), (1, 0.25, 4), (2, 0.3, 7)],
+    [(1, 1.0, 1), (2, 1.0, 2), (1, 0.25, 4), (2, 0.3, 6)],
```

Comment:
`save_last` is not set True, therefore we have on less checkpoint?

Reply:
Can we parametrize `save_last=True` as we should have the same number of checkpoint as previously generated.

Reply:
> save_last is not set True, therefore we have on less checkpoint?

Correct

> Can we parametrize save_last=True as we should have the same number of checkpoint as previously generated.

Will do, but this increases the expected count for all cases. The difference here is due to an implementation bug when the number of batches does not align with the validation check interval requested (see my other comment below)